pkg/kubelet/container/runtime.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"github.com/fsouza/go-dockerclient"
+)
+
+// ContainerRuntimeInterface is the abstract interface for container runtime.
+type ContainerRuntimeInterface interface {
+	ListContainers(options docker.ListContainersOptions) ([]docker.APIContainers, error)
```

Comment:
There are a number of docker-specific arguments here. Any ideas about what to do about that?